### DISCUSSION TITLE
Please share more tutorials

### DISCUSSION THREAD

**Discussion by Freestyleyang** [pyswmm/pyswmm#491] [Ideas]
Hi there, thank you for your great work! I find the Chapter 4 tutorial "Extracting Results and Plotting" extremely helpful, however, only 4.1 and 4.2 are currently available. Is it possible to update videos for 4.3, 4.4, 4.5, and 4.6? Thank you and have a great one!

[https://www.pyswmm.org/tutorial#h.z5vfp7xfmece](url)

**Comment by bemcdonnell** [comment]
@Freestyleyang, thanks for the support! I’ll see what I can do soon to record, cut, and upload these videos.  Until then feel free to check out our little pyswmm swag store 😎😎

https://www.zazzle.com/store/pyswmm

**Comment by Freestyleyang** [comment]
@bemcdonnell Thanks a lot for your reply. I greatly look forward to the tutorial! Please have some T-shirts in the pyswmm store, that will be amazing! 

In the meantime, I am trying to calibrate a SWMM model using PySWMM, however, I am not sure how I can change or manipulate SWMM parameters using PySWMM. Could you direct me to some materials that I can use? Thanks again!
